instagram/src/App.css

```diff
@@ -0,0 +1,8 @@
+*{
+    box-sizing: border-box;
+}
+
+.App{
+    width: 1024px;
```

Comment:
You are hard coding a width for the whole app which is not make your site responsive